### ISSUE TITLE
Potentials list shows impossible colours

### ISSUE THREAD

**Issue by teptec** [canonn-science/canonn-signals#2]
Here's an opportunity to improve biological predictions. 
Predictions were Fonticula Campestris - Amethyst or Emerald or Mauve or Yellow.

![image](https://user-images.githubusercontent.com/96891024/153706950-58c4592f-ac07-4f90-bfb7-8a483c381912.png)

But this body orbits a Class L dwarf:

![image](https://user-images.githubusercontent.com/96891024/153706964-c6921615-cf94-4b52-b541-436c6b98d367.png)

So the colour must be Mauve, and indeed:

![image](https://user-images.githubusercontent.com/96891024/153706979-55a3e148-6346-4f29-aaff-66df5d4017c4.png)

Here's a simpler one. There's no way this Fonticulua could have been Amethyst.

![image](https://user-images.githubusercontent.com/96891024/153707017-713c8901-0544-4f13-ba9f-d67f344d8ad1.png)

![image](https://user-images.githubusercontent.com/96891024/153707000-79b521e9-20fe-433d-841e-5c6da6824905.png)

![image](https://user-images.githubusercontent.com/96891024/153707006-20fcbff5-4fbb-4f3f-a085-8c046480e457.png)

Ensuing discussion from #chat_geology_biology:
Teptec:
Or maybe it could have been Amethyst due to the main star being a Class M.
But it couldn't have been Emerald or Yellow as these require the presence of a K or an F class star.

Felsen[OBT] — Today at 09:56
its not the main star that counts in the usual case ... if the star the planet orbits is counted as such and not as a planet by ED, this will determine the color

Scopelx [OBT] — Today at 09:57
I'm annoyed that bios on moons of stars in planetary orbits seem to default to the primary star's variety
Gonna make the Y flavors a pain

Felsen[OBT] — Today at 09:58
yeah, thats the case where ED counts them as planets and not as stars

Teptec — Today at 09:58
Indeed. In this case I was almost certain it would be Mauve. But there are edge cases. e.g. Fonticulua Campestris - Yellow on Blia Chria BA-A d43 B 3 (according to the spreadsheet, I haven't confirmed by eye)
Blia Chria BA-A d43 B may be considered a planet.

Felsen[OBT] — Today at 10:02
when it has a letter rather than a number it should usually be considered a star ... but yeah, the engine does have its quirks, so ...

**Comment by NoFoolLikeOne** [comment]
Yes, the problem here for me is that the naming convention determines the star class but the stats gathered is not clear. It should say must have so perhaps we have some duff data somewhere

**Comment by NoFoolLikeOne** [comment]
When you give examples could you include a list to the signals page and then I can easily go and look at the data

**Comment by NoFoolLikeOne** [comment]
https://canonn-science.github.io/canonn-signals/index.html?system=Eor%20Aed%20BK-G%20c24-1

**Comment by NoFoolLikeOne** [comment]
Thanks @teptec that looks better now. I'm using the internal name to determine the star class and excluding if they don't match. But we still need to find out why we have some recorded with the wrong class. I'll get the spansh parser to spit out any mismatches

**Comment by teptec** [comment]
I have a few systems flagged from my analysis of the data available in the Biologicals spreadsheet and pulled down from the API.
I'm heading back to the Bubble where I'll do some admin, then I'll map a course through some of these systems which appear to have bogus data and see what's actually going on.

**Comment by teptec** [comment]
FYI as a follow up to my last comment. 
When I returned to the Bubble I did indeed visit about 20 systems which had very unusual observations in the Google Sheet.
i.e. one system with gravity 0.2 with a particular biological observed then gravity on the next lowest-gravity  body being 0.4,
All the observations checked out, i.e. the  biological was present and the body parameters were as per the spreadsheet.


I was actually very disappointed not to be able to find any bad data in the Google sheet, I was sure a couple of those records would prove to be bogus.
